packages/block-editor/src/components/use-block-display-information/index.js

```diff
@@ -67,21 +67,30 @@ export default function useBlockDisplayInformation( clientId ) {
 	return useSelect(
 		( select ) => {
 			if ( ! clientId ) return null;
-			const { getBlockName, getBlockAttributes } =
-				select( blockEditorStore );
+			const {
+				getBlockName,
+				getBlockAttributes,
+				__experimentalGetReusableBlockTitle,
+			} = select( blockEditorStore );
 			const { getBlockType, getActiveBlockVariation } =
 				select( blocksStore );
 			const blockName = getBlockName( clientId );
 			const blockType = getBlockType( blockName );
 			if ( ! blockType ) return null;
+			let title = blockType.title;
 			const attributes = getBlockAttributes( clientId );
 			const match = getActiveBlockVariation( blockName, attributes );
-			const isSynced =
-				isReusableBlock( blockType ) || isTemplatePart( blockType );
+			const isReusable = isReusableBlock( blockType );
+			if ( isReusable ) {
```

Comment:
Totally optional, but maybe this condition could be rejigged a bit. 

`blockType.title` is already assigned to the `title` var.

What about 
```js
const resusableTitle = isReusableBlock( blockType ) ? __experimentalGetReusableBlockTitle( attributes.ref ) : '';
const title = resusableTitle || blockType?.title;
```

Reply:
Seemed like the ternary needed to be set to `undefined` not `''` if not reusable, but maybe I am missing something, eg.

```js
const resusableTitle = isReusable ? __experimentalGetReusableBlockTitle( attributes.ref ) : undefined;
const title = resusableTitle || blockType.title;
```

also on line 79 the function returns early if `blockType` is not defined, so the optional chaining is not need for blockType.title - but have otherwise made that change as it is a bit tidier if you want to double check it @ramonjd

Reply:
> Seemed like the ternary needed to be set to undefined not '' if not reusable, but maybe I am missing something

Sounds good. My example was "unrehearsed". 

I retested and it works as advertised. Thanks @glendaviesnz !